File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/RoomFinderSuggestionProvider.java
package de.tum.in.tumcampusapp.component.tumui.roomfinder;

import android.content.SearchRecentSuggestionsProvider;

/**
 * Suggestion provider for {@link RoomFinderFragment}
 */
public class RoomFinderSuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.roomfinder.RoomFinderSuggestionProvider";

    public RoomFinderSuggestionProvider() {
        setupSuggestions(AUTHORITY, 1);
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/RoomFinderSuggestionProvider.kt
package de.tum.`in`.tumcampusapp.component.tumui.roomfinder

import android.content.SearchRecentSuggestionsProvider

/**
 * Suggestion provider for [RoomFinderActivity]
 */
class RoomFinderSuggestionProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(AUTHORITY, 1)
    }

    companion object {
        const val AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.roomfinder.RoomFinderSuggestionProvider"
    }
}